### PR TITLE
[patch] Remove redundant conversion of project name

### PR DESCRIPTION
**Pull request by samwaseda**
Trying to make smaller PRs to make [this PR](https://github.com/pyiron/pyiron_base/pull/1440) lighter. I guess this one is a small patch

### CODE REVIEW

**Comment by jan-janssen** [comment]
Can we add unit tests for the `pack()` and `unpack()` to make sure we get it right this time.

**Comment by samwaseda** [comment]
> Can we add unit tests for the `pack()` and `unpack()` to make sure we get it right this time.

There are already unit tests and right now I'm not adding new functionalities. When I come to the point where I start adding something I'll add unit tests.

**Comment by samwaseda** [comment]
> Can we add unit tests for the `pack()` and `unpack()` to make sure we get it right this time.

I added assertion tests in the code, because I just don't understand what kind of situation was imagined previously.

File: pyiron_base/project/archiving/export_archive.py [hunk 1]
```diff
@@ -151,7 +130,7 @@ def export_database(project_instance, directory_to_transfer, archive_directory):
         for job_id in df.parentid
     ]
     df["project"] = update_project(
-        project_instance,
+        pr,
```
Comment (jan-janssen) [inline pyiron_base/project/archiving/export_archive.py]:
```suggestion
        project_instance=pr,
```

**Review by jan-janssen — APPROVED** [review]
Looks good to me

**Pull request merged**